<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game_view</class>
 <widget class="QWidget" name="game_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Timer_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;8BIT WONDER&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>541</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="timer_text">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;8BIT WONDER&quot;;
color: rgb(255, 255, 255);
background-color: rgb(48, 48, 48);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>811</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/assets/background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>imageLabel</zorder>
  <zorder>Timer_label</zorder>
  <zorder>pushButton</zorder>
  <zorder>graphicsView</zorder>
  <zorder>timer_text</zorder>
  <zorder>back_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
